meson.build

```diff
@@ -15,11 +18,14 @@ add_project_arguments(['--vapidir', join_paths(meson.current_source_dir(), 'vapi
 
 conf = configuration_data()
 conf.set_quoted('GETTEXT_PACKAGE', meson.project_name())
-configure_file(output: 'config.h', configuration: conf)
+configure_file(
+    output: 'config.h',
+    configuration: conf
+)
```

Comment:
Same here. This code style change should be in a code style PR. Not related to i18n changes